lib/MongoDB/MongoClient.pm

```diff
@@ -1440,6 +1530,7 @@ sub send_admin_command {
         query_flags => {},
         bson_codec  => $self->bson_codec,
         read_preference => $read_pref,
+        client      => $self,
```

Comment:
Also, add a comment on this function that this doesn't use sessions -- which I think is correct.